homeassistant/components/thermostat/homematic.py

```diff
@@ -7,42 +7,102 @@
 import logging
 import socket
 from xmlrpc.client import ServerProxy
+from xmlrpc.client import Error
 
 from homeassistant.components.thermostat import ThermostatDevice
 from homeassistant.const import TEMP_CELCIUS
+from homeassistant.helpers.temperature import convert
 
 REQUIREMENTS = []
 
+_LOGGER = logging.getLogger(__name__)
+
 CONF_ADDRESS = 'address'
 CONF_DEVICES = 'devices'
 CONF_ID = 'id'
 PROPERTY_SET_TEMPERATURE = 'SET_TEMPERATURE'
 PROPERTY_VALVE_STATE = 'VALVE_STATE'
 PROPERTY_ACTUAL_TEMPERATURE = 'ACTUAL_TEMPERATURE'
 PROPERTY_BATTERY_STATE = 'BATTERY_STATE'
+PROPERTY_LOWBAT = 'LOWBAT'
 PROPERTY_CONTROL_MODE = 'CONTROL_MODE'
+PROPERTY_BURST_MODE = 'BURST_RX'
+TYPE_HM_THERMOSTAT = 'HOMEMATIC_THERMOSTAT'
+TYPE_HM_WALLTHERMOSTAT = 'HOMEMATIC_WALLTHERMOSTAT'
+TYPE_MAX_THERMOSTAT = 'MAX_THERMOSTAT'
 
-_LOGGER = logging.getLogger(__name__)
+
+class HomematicConfig():
```

Comment:
Please use a named tuple. It will produce exactly the same object.
```python
from collections import namedtuple
HomematicConfig = namedtuple('HomematicConfig', ['device_type', 'platform_type', 'channel', 'maint_channel'])
```